bridge/Dockerfile

```diff
@@ -26,6 +26,13 @@ ENV VERSION="${version}"
 ENV API_URL "http://api-gateway-nginx.keptn.svc.cluster.local"
 ENV API_TOKEN ""
 
+# OAuth and session cookie related configurations
```

Comment:
We should move this ENV-variables into deploy/bridge.yaml and the Keptn installer Helm-chart.

Reply:
Thank you @agrimmer . I fixed this with the latest commit.